2gp-demo/force-app/main/default/classes/LicensingUtilTest.cls

```diff
@@ -10,11 +10,6 @@ private class LicensingUtilTest {
         String result = LicensingUtil.getPackageId();
     }
 
-    @isTest
-    static void testIsPackagingOrg() {
-        Boolean result = LicensingUtil.isPackagingOrg();
-    }
-
     @isTest
     static void testUserHasPackageLicense() {
```

Comment:
this class was mainly for code coverage I believe. should there be any real asserts here for real testing, or is that too complicated given the various running contexts?

Reply:
Adding real asserts is complicated. May be not worth spending time on that.

Reply:
sounds good